servers/zts/src/main/java/com/yahoo/athenz/zts/ZTSConsts.java

```diff
@@ -192,4 +192,6 @@
     public static final String ZTS_CERT_RECORD_STORE_FACTORY_CLASS = "com.yahoo.athenz.zts.cert.impl.FileCertRecordStoreFactory";
 
     public static final String ZTS_JSON_PARSER_ERROR_RESPONSE = "{\"code\":400,\"message\":\"Invalid Object: checkout https://github.com/yahoo/athenz/core/zts for object defintions\"}";
+    public static final String ZTS_NUMBER_OF_THREADS_S3_BUCKET = "10";
```

Comment:
don't need this. for just default values, just include 10 in the getProperty call.